### PR TITLE
Thematic update for Canada Day

### PR DESCRIPTION
**Pull request by nfrenette**
Updated red hex and added beige CSS and examples

### CODE REVIEW

File: méli-mélo/th-canadaday/style.css [hunk 1]
```diff
@@ -1,33 +1,59 @@
...
 a.bg-canadaday:hover,
 a.bg-canadaday:focus,
 a.bg-canadaday:active {
-    color: #fff;
+    color: #fff !important;
+}
+    
+/* Thematic beige */
+.bg-canadaday-beige,
+body .bg-canadaday-beige {
```
Comment (duboisp) [inline méli-mélo/th-canadaday/style.css:37]: Need a demo or be removed.

With gcweb-jekyll I do think you can use the front-matter "pageclass" like our pink-day working example demo

**Comment by GormFrank** [comment]
Pre-approved during technical review upon successful testing. (NEED TO SQUASH AND MERGE)

**Review by duboisp — CHANGES_REQUESTED** [review]
All CSS need to be demoed.

File: méli-mélo/th-canadaday/canada-day.html [hunk 2]
```diff
@@ -50,40 +56,40 @@ <h3>Code</h3>
...
+&lt;/section&gt;</code></pre>
+    <h3>&lt;button&gt; .btn example</h3>
+    <button class="btn btn-info bg-canadaday text-white" type="button">Button</button>
     <h4>Code</h4>
-    <pre><code>&lt;button type="button" class="bg-dark bg-canadaday text-white"&gt;Button&lt;/button&gt;</code></pre>
-    <h3>&lt;a&gt; btn example</h3>
-    <a href="#" class="btn btn-primary bg-canadaday text-white" role="button">Button</a>
+    <pre><code>&lt;button class="btn btn-info bg-canadaday text-white" type="button"&gt;Button&lt;/button&gt;</code></pre>
+    <h3>&lt;a&gt; .btn example</h3>
+    <a href="#" class="btn btn-info bg-canadaday text-white" role="button">Button</a>
```
Comment (duboisp) [inline méli-mélo/th-canadaday/canada-day.html:85]:
```suggestion
    <a href="#" class="btn btn-info bg-canadaday text-white">Anchor</a>
```

File: méli-mélo/th-canadaday/canada-day.html [hunk 3]
```diff
@@ -50,40 +56,40 @@ <h3>Code</h3>
...
     <h4>Code</h4>
-    <pre><code>&lt;a href="#" class="btn btn-primary bg-canadaday text-white" role="button"&gt;Button&lt;/a&gt;</code></pre>
+    <pre><code>&lt;a href="#" class="btn btn-info bg-canadaday text-white" role="button"&gt;Button&lt;/a&gt;</code></pre>
```
Comment (duboisp) [inline méli-mélo/th-canadaday/canada-day.html:87]:
```suggestion
    <pre><code>&lt;a href="#" class="btn btn-info bg-canadaday text-white"&gt;Anchor&lt;/a&gt;</code></pre>
```

File: méli-mélo/th-canadaday/canada-day.html [hunk 4]
```diff
@@ -97,10 +103,68 @@ <h4>Code</h4>
...
+    &lt;p&gt;This section is styled with &lt;code&gt;.bg-canadaday-beige&lt;/code&gt;.&lt;/p&gt;
+    &lt;p&gt;&lt;a href="#"&gt;Link example&lt;/a&gt;&lt;/p&gt;
+    &lt;button class="btn-default bg-canadaday-beige" type="button"&gt;Button example&lt;/button&gt;
+&lt;/section&gt;</code></pre>
+    <h3>&lt;button&gt; .btn example</h3>
+    <button class="btn btn-default bg-canadaday-beige" type="button">Button</button>
+    <h4>Code</h4>
+    <pre><code>&lt;button class="btn btn-default bg-canadaday-beige" type="button"&gt;Button&lt;/button&gt;</code></pre>
+    <h3>&lt;a&gt; .btn example</h3>
+    <a href="#" class="btn btn-default bg-canadaday-beige" role="button">Button</a>
```
Comment (duboisp) [inline méli-mélo/th-canadaday/canada-day.html:167]:
```suggestion
    <a href="#" class="btn btn-default bg-canadaday-beige">Anchor</a>
```

File: méli-mélo/th-canadaday/canada-day.html [hunk 5]
```diff
@@ -97,10 +103,68 @@ <h4>Code</h4>
...
+    <h4>Code</h4>
+    <pre><code>&lt;a href="#" class="btn btn-default bg-canadaday-beige" role="button"&gt;Button&lt;/a&gt;</code></pre>
```
Comment (duboisp) [inline méli-mélo/th-canadaday/canada-day.html:169]:
```suggestion
    <pre><code>&lt;a href="#" class="btn btn-default bg-canadaday-beige"&gt;Anchor&lt;/a&gt;</code></pre>      
```

**Review by duboisp — APPROVED** [review]
All examples are demoed, documented and will nicely fallback according to the progressive enhancement design principle.

My few change request is to improve the anchor working example. An anchor should not have the role button when it is not acting as a button.  Also, when a role is available in the host language, we should use the host language before to override it with ARIA.

I know that you have enforced the text-white on link to avoid the little blue but I remarked the button still show the little blue on mouse hover.

**Pull request merged**